docs/documentation/mediaObserver.md

```diff
@@ -0,0 +1,151 @@
+The injectable **MediaObserver** service will provide mediaQuery **activations** notifications for all
+[registered BreakPoints](https://github.com/angular/flex-layout/wiki/Custom-Breakpoints).
+
+This service is essentially an Observable that exposes both features to subscribe to mediaQuery
+changes and a validator method `.isActive()` to test if a mediaQuery (or alias) is
+currently active.
+
+> Only mediaChange activations (not deactivations) are announced by the MediaObserver
+
+---
+
+### API Summary
+
+The injectable **MediaObserver** service has two (2) APIs:
+
+- `asObservable(): Observable<MediaChange>`
+- `isActive(query: string): boolean`
+
+---
+
+#### 1. **`MediaObserver::asObservable`**
+
+Developers use Angular DI to inject a reference to the **MediaObserver** service as a **constructor** parameter.
+
+The **MediaObserver** is not an actual **Observable**. It is a wrapper of an Observable used to publish additional methods like `isActive(<alias>).
+
+```typescript
+asObservable(): Observable<MediaChange>
+```
+
+Use the `.asObservable()` accessor method to access the **Observable** and use **RxJS** operators; such as `media.asObservable().filter(....)`.
+
+> Do not forget to **import** the specific RxJS operators you wish to use!
+
+```typescript
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { MediaChange, MediaObserver } from "@angular/flex-layout";
+import { Subscription } from "rxjs";
+import { distinctUntilChanged } from "rxjs/operators";
+
+@Component({
+  selector: "app-root",
+  templateUrl: "./app.component.html",
+  styleUrls: ["./app.component.css"],
+})
+export class AppComponent implements OnInit, OnDestroy {
+  title = "Angular Flex-Layout";
+
+  constructor(private mediaObserver: MediaObserver) {}
+  private mediaSubscription!: Subscription;
+  private activeMediaQuery: string = "";
```

Comment:
Redundant type parameter

Reply:
I assume this is the title? sorry I meant to remove that before doing the PR

Reply:
No, I mean `string` is unnecessary when you are assigning it to the empty string.